oap-ape/ape-java/ape-common/src/main/java/com/intel/ape/service/netty/NettyMessage.java

```diff
@@ -648,7 +651,7 @@ private void writeDataToChannel(ChannelOutboundInvoker out, ChannelPromise promi
                                         ByteBufAllocator allocator, int headerLength,
                                         int originLength) {
             writeHeaderBufToChannel(out, promise, allocator, headerLength,
-                    originLength, originLength, false);
+                    originLength, originLength, false, null);

```

Comment:
safe to use null? use empty string instead?

Reply:
this value will be only used after compressEnabled checked. so it's okay to give null here.